gulpfile.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('test', function(){
+  gulp.src('./test/*-test.js',{read:false})
+  .pipe(mocha({reporter:'spec'}));
+});//end test task
+
+gulp.task('lint', function(){
+  return gulp.src(['**/*.js', '!node_modules'])
+  .pipe(eslint())
+  .pipe(eslint.format())
+  .pipe(eslint.failAfterError());
+});
+
+gulp.task('default', ['test', 'lint']);
```

Comment:
@khalidM3 @reganoneill @dkulp23 `gulpfile.js` looks good - be sure to add additional tasks, as needed, for your own personal development workflow